StripeIdentity/StripeIdentity/Source/API Bindings/DocumentScanner+API.swift

```diff
@@ -11,6 +11,9 @@ import Foundation
 import Vision
 
 extension DocumentScanner.Configuration {
+    // TODO: collect historical data and update the threshold from server.
+    static let defaultBlurThreshold: Decimal = 0.0
```

Comment:
currently the default value is set to 0, so any number will pass. Different threshold will be sent from server through experiments later